jobs/ClusterHQ_Senior_Lead_QA

```diff
@@ -0,0 +1,74 @@
+title: Senior / Lead Software Test Engineer
+company: ClusterHQ
+url: http://techfolk.co.uk/current-jobs/senior-lead-software-test-engineer-central-bristol-tl103
+location: Bristol, England
+contract: permanent
+contact:
+    name: Andrew Gifford
+    email: hello@techfolk.co.uk
+    phone: +44 (0)117 318 2447
+    ...: ...
+created: 2015-08-04
+tags:
+  - Bristol
+  - England
+  - uk
+  - relocation
+  - QA
+  - testing
+  - software testing
+  - containers 
+  - containerisation
+  - containerization
+  - virtualisation
+  - virtualization
```

Comment:
I don't think we should duplicate all the tags just to handle US English. I'd be happy to go with British English spellings.